guides/common/modules/proc_overriding-ansible-variables-in-project.adoc

```diff
@@ -48,23 +48,9 @@ If the hashes contain the same key, the value is overwritten by the value of the
 . Optional: Expand the *Specify Matchers* area and specify criteria for selecting hosts on which the variable overrides.
 . To save the override settings, click *Submit*.
 
-To use the Ansible variable, add the variable as a parameter to your host or host group, or add the variable as a global parameter.
-
-.Adding the variable to a host
-. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and select the host that you want to use.
-. Click the *Ansible* tab, and in the *Variables* area, click the pencil icon to edit the value of the variable.
+.Overriding variables for hosts
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Select your host.
+. On the *Ansible* tab, select the *Variables* tab.
+. Click the pencil icon to edit the value of the variable.
 . Click the tick icon to accept the value of the changed variable or the cross icon to cancel the change.
-
-.Adding the variable to a host group
```

Comment:
what are the reasons to delete these sections?

Reply:
At the time, I could not reproduce this. I can reproduce it by now, so looks like my changes are wrong.